### PR TITLE
IO1_3027: If total amount is less than approved allowance we need to approve new spending caps

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

File: src/app/components/applications/SSV/MyAccount/components/ReactivateCluster/ReactivateCluster.tsx [hunk 1]
```diff
@@ -13,13 +13,13 @@ import ErrorMessage from '~app/components/common/ErrorMessage';
...
 import FundingSummary from '~app/components/common/FundingSummary';
 import SsvStore from '~app/common/stores/applications/SsvWeb/SSV.store';
-import PrimaryButton from '~app/components/common/Button/PrimaryButton';
 import { formatNumberToUi, propertyCostByPeriod } from '~lib/utils/numbers';
 import WalletStore from '~app/common/stores/applications/SsvWeb/Wallet.store';
 import ValidatorStore from '~app/common/stores/applications/SsvWeb/Validator.store';
 import NewWhiteWrapper from '~app/components/common/NewWhiteWrapper/NewWhiteWrapper';
 import ApplicationStore from '~app/common/stores/applications/SsvWeb/Application.store';
 import ProcessStore, { SingleCluster } from '~app/common/stores/applications/SsvWeb/Process.store';
+import Button from '~app/components/common/Button/Button';
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/components/ReactivateCluster/ReactivateCluster.tsx:22]: Move this import upper according to good formatting rule

File: src/app/components/applications/SSV/MyAccount/components/ReactivateCluster/ReactivateCluster.tsx [hunk 2]
```diff
@@ -139,8 +139,13 @@ const ReactivateCluster = () => {
                   <Typography className={classes.SsvPrice}
                               style={{ marginBottom: 0 }}>{formatNumberToUi(totalCost.toFixed(18))} SSV</Typography>
                 </Grid>
-                <PrimaryButton text={'Next'} submitFunction={reactivateCluster}
-                               disable={insufficientBalance || customPeriod <= 0 || isNaN(customPeriod)}/>
+                <Button
+                    withAllowance
+                    text={'Next'}
+                    onClick={reactivateCluster}
+                    disable={insufficientBalance || customPeriod <= 0 || isNaN(customPeriod)}
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/components/ReactivateCluster/ReactivateCluster.tsx:146]: Move this complex condition to the same section of component where all the rest of calculations/flags defined and use here only 1 variable

**Review by meshin-blox — CHANGES_REQUESTED** [review]

File: src/app/components/applications/SSV/MyAccount/components/ReactivateCluster/ReactivateCluster.tsx (hunk 1 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/ReactivateCluster/ReactivateCluster.tsx:22]: done

File: src/app/components/applications/SSV/MyAccount/components/ReactivateCluster/ReactivateCluster.tsx (hunk 2 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/ReactivateCluster/ReactivateCluster.tsx:146]: done

**Review by meshin-blox — APPROVED** [review]

**Pull request merged**